shed/event_streams.py

```diff
@@ -1110,22 +1111,31 @@ class Eventify(EventStream):
     >>> assert len(L) == 4
     """
 
-    def __init__(self, child, start_key, *, output_info, **kwargs):
+    def __init__(self, child, *start_keys, output_info, **kwargs):
```

Comment:
I guess this means we need to make the `output_info` optional or allow it to be None, since we don't know what key/values may live inside the start itself.